llvm/test/Transforms/InstCombine/AMDGPU/mbcnt-to-bitmask-neg.ll

```diff
@@ -0,0 +1,18 @@
+; RUN: opt -S -mtriple=amdgcn-unknown-amdhsa -passes=instcombine < %s | FileCheck %s
+; CHECK-NOT: and i32
+; CHECK-NOT: @llvm.amdgcn.workitem.id.x()
```

Comment:
Generate full checks and avoid using CHECK-NOT

Reply:
Done